README.md

```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 <div align="center">
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Vet Clinic</b></h3>
 
 </div>
 
```

Comment:
- [ ] Kindly include a valid LICENSE link to the LICENSE section in your README file. When I click the link, it displays a 404 error page. Adding an open-source license to your repository ensures that others can use, copy, modify, and contribute back to your project. It's an important step when creating an open-source project. 😎